### ISSUE TITLE
.nuke after commands don't run on the server

### ISSUE THREAD

**Issue by Krutoy-4el** [TKperson/Nuking-Discord-Server-Bot-Nuke-Bot#73]
**Describe the bug**
When running .nuke on the server, after commands don't run.

**To Reproduce**
1. Start the bot
2. Add after commands
3. Run .nuke directly on the server you want to nuke
4. After commands do not run

**PROBLEM**
I've debugged a little and found the mistake.
Here it is (code of `async def log`):
 
    try:
        await ctx.send(message)
    except discord.errors.HTTPException:
        for i in range(ceil(len(message) / 2000)):
            await log(ctx, message[2000 * i:2000 * (i + 1)])
    except:
        consoleLog(message)

The reason is - NOT every HHTPException is caused by a lot of text, and it catches `NotFound` error too (`issubclass(discord.NotFound, discord.HTTPException)` returns `True`). Because of this, when channel where the command was ran is deleted, it keeps calling log again and again.
In fact HTTPException always has code of exception, described on [Discord Developers webpage](https://discord.com/developers/docs/topics/opcodes-and-status-codes). For exceeding the limit of 2000 characters it is 50035 - Invalid form body.
So may be it's worth to check the error code to avoid such unexpected behavior?

**Comment by TKperson** [comment]
Yes, I do see the potential of this being recursive, and the `discord.NotFound` is a subclass of `discord.HTTPException` and the `discord.NotFound` error can be caught within the `discord.HTTPException`. At first I didn't really thought it was going to cause a problem until I read through[this](https://github.com/Rapptz/discord.py/blob/master/discord/errors.py). I wasn't as familier with discord.py before and missed the potential of this recursive problem.

Also the `await log(ctx, message[...])` is supposed to be `await ctx.send(message[...])`. I'm going to fix these asap. Preferably, this patch should get released along with 2.5 update
